command/ca/provisioner/provisioner.go

```diff
@@ -368,6 +368,32 @@ Use the flag multiple times to remove multiple challenges.`,
 If this flag is set to false, then disable EAB.`,
 	}
 
+	attestationFormatFlag = cli.StringSliceFlag{
+		Name: "attestation-format",
+		Usage: `Enable an ACME attestation statement <format> in the provisioner. Use the flag
+multiple times to configure multiple challenges.
+
+The supported ACME attestation formats are:
+
+**apple**
+: With the apple format, Apple devices can use the device-attest-01 challenge to
+get a new certificate.
+
+**step**
```

Comment:
Why do we call this `step`?

Reply:
We need a custom format to differentiate the different requests, apple has its own. 

The format `step` is based on the attestation format `packed` defined in https://www.w3.org/TR/2019/REC-webauthn-1-20190304/#packed-attestation, but the verification of the attestation certificates is slightly different.

Reply:
We can switch to `packed` the format is exactly the same. Just the verification is different.